test/src/module/config/parseTest.c

```diff
@@ -87,7 +87,7 @@ testRun(void)
         // -------------------------------------------------------------------------------------------------------------------------
         TEST_TITLE("config command defaults to none before cfgInit()");
 
-        TEST_RESULT_UINT(cfgCommand(), cfgCmdNone, "command is none");
+        TEST_RESULT_UINT(cfgCommand(), cfgCmdHelp, "command is helps");
```

Comment:
typo? `helps` -> `help`

Reply:
Whoops -- I fixed that and a few other comments but forgot to push it. Fixed in cd5eec846.